app/controllers/admin_controller.rb

```diff
@@ -33,8 +33,9 @@ def articles
   end
 
   def update_order
-    object = params[:object].titleize.constantize
-    @obj = object.find(params[:obj_id])
+    # Safely identify the model we're updating the position of
+    klass = [Category, Doc].detect { |c| c.name.casecmp(params[:object]) == 0 }
```

Comment:
That is correct as of that update.  I think there may be value in adding it to forums as well (just the forum display order- not topics) but for now lets stick with just Doc and Cat.